src/osemgrep/language_server/Diagnostics.mli

```diff
@@ -1,10 +1,12 @@
 val diagnostics_of_results :
+  bool ->
```

Comment:
this is one of the few cases where I find labels useful, so I would instead
have an is_intellij: bool in the signature

Reply:
fair